public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -77,13 +96,32 @@ function ContextMenu (uiContextMenu) {
     }
 
     function handleCloseButtonClick () {
+
         svl.tracker.push('ContextMenu_CloseButtonClick');
         hide();
+        _handleSeverityPopup();
+
     }
 
     function _handleOKButtonClick () {
+
         svl.tracker.push('ContextMenu_OKButtonClick');
         hide();
+        _handleSeverityPopup();
+
+    }
+
+    function _handleSeverityPopup (){
+        var labels = svl.labelContainer.getCurrentLabels();
+        var prev_labels = svl.labelContainer.getPreviousLabels();
+        if (labels.length == 0){
+            labels = prev_labels;
+        }
+        if (labels.length > 0) {
```

Comment:
The reason was that the previous labels could also have a 0 length, which is accounted for by the second if statement.